ui/app/src/Views/SilenceForm/Updates.elm

```diff
@@ -116,7 +117,7 @@ update : SilenceFormMsg -> Model -> ( Model, Cmd Msg )
 update msg model =
     case msg of
         CreateSilence silence ->
-            ( model
+            ( initSilenceForm
```

Comment:
Is this necessary? The creation of the silence will send the user through the `SilenceCreate` message path, which sets the model to either `Failure` or `Success`. I'm wondering what you were seeing that prompted this PR.